be/src/storage/inverted/clucene/clucene_inverted_writer.cpp

```diff
@@ -224,6 +230,7 @@ class CLuceneInvertedWriterImpl : public CLuceneInvertedWriter {
     InvertedIndexParserType _parser_type;
     std::wstring _field_name;
     const TabletIndex* _inverted_index;
+    std::wstring _wstring_buffer;
```

Comment:
if wstring_buffer is just used for a continuous buffer, shall use raw byte or raw char array instead of wstring, to avoid the overhead of initializing the string.

Reply:
Use raw byte buffer instead.